class NameAlreadyTakenException(Exception):
    pass


class GameObjectNotFoundException(Exception):
    pass
